vnext/src/Libraries/AppTheme/AppTheme.ts

```diff
@@ -5,6 +5,7 @@
  */
 'use strict';
```

Comment:
>strict [](start = 5, length = 6)

So, this isn't directly related to 0.62, but I discovered when cleaning up our flowconfig that we forgot to remove the stub variant of AppTheme when we removed the stubs of other Windows specific JS. Fixed that here and removed the stub.